### ISSUE TITLE
\graphicspath{ {dir1} {dir2} } not properly detected because of the white spaces

### ISSUE THREAD

**Issue by xq114** [James-Yu/LaTeX-Workshop#1853]
When I insert a graph to the file I'm delighted to find an autocomplete suggestion about files in the current folder. However, I'd like to put graphs in another folder and use `\graphicspath{}` command to find them. I'm really in need for a detection of the folders inside command `\graphicspath{}`.
![image](https://user-images.githubusercontent.com/42310255/70126575-4d98b800-16b4-11ea-8a95-04c27fa3032d.png)

**Comment by jlelong** [comment]
Completion for `\includegraphics` is taking `\graphicspath` into account. I cannot reproduce your issue.

**Comment by xq114** [comment]
I've made a range of tests and found something really confusing. It seems like a bug.
The first two results were all right:
![result1](https://user-images.githubusercontent.com/42310255/70140505-ceb17880-16cf-11ea-98ae-05aee2594148.png)
![result2](https://user-images.githubusercontent.com/42310255/70140532-e38e0c00-16cf-11ea-95cb-3d962fb57fa5.png)
but when I changed the folder inside the command `\graphicspath{}` the intellisense did not change:
![result3](https://user-images.githubusercontent.com/42310255/70140666-323ba600-16d0-11ea-9448-a1cade330507.png)
and I found it even more confusing when I turned to the former tex file:
![result4](https://user-images.githubusercontent.com/42310255/70140738-5eefbd80-16d0-11ea-975b-ffefd4dfa5b8.png)


Here list some of my version info.
version: 1.40.2 (user setup)
commit: f359dd69833dd8800b54d458f6d37ab7c78df520
date: 2019-11-25T14:54:45.096Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18362

LaTeX Workshop Version: 8.4.2
